src/styles/table.styl

```diff
@@ -32,9 +32,13 @@ $set-device
         flex 0 0 15%
 
     .set-table-devices
+        height auto!important //@stylint ignore
```

Comment:
We must not use `!important`. Why can't we avoid using it here ?

Reply:
Actually this is a quickfix for cozy-ui, I forget to put a comment here but that will be removed as soon as tables in cozy-ui are ready.